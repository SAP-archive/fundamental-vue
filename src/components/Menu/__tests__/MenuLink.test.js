import { mount } from "@vue/test-utils";
import FdMenuLink from "./../MenuLink.vue";

describe("MenuLink", () => {
  it("renders correctly", () => {
    expect(
      mount({
        template: `<FdMenuLink>Item 1</FdMenuLink>`,
        components: { FdMenuLink }
      }).element
    ).toMatchSnapshot();
  });
});
