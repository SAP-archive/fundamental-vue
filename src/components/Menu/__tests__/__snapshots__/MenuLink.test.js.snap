// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuLink can be disabled 1`] = `
<a
  class="fd-menu__item is-disabled fd-has-color-text-4"
  data-fd-menu-item-link-uid="aaa"
>
  title
</a>
`;

exports[`MenuLink renders correctly 1`] = `
<a
  class="fd-menu__item"
  data-fd-menu-item-link-uid="aaa"
>
  Item 1
</a>
`;
